modules/distribution/product/src/main/conf/synapse-configs/default/inbound-endpoints/SecureWebhookServer.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<inboundEndpoint name="SecureWebhookServer"
+                 protocol="https"
+                 suspend="false" sequence="TestIn" onError="fault" >
+    <p:parameters xmlns:p="http://ws.apache.org/ns/synapse">
+        <p:parameter  name="inbound.http.port">8021</p:parameter>
+        <p:parameter name="keystore">
+            <KeyStore>
+                <Location>repository/resources/security/wso2carbon.jks</Location>
```

Comment:
if this key stores are hardcoded will become impact on changing this can we add templated file

Reply:
+1. But we have done the same for other inbound endpoints.

Reply:
yes